tests/src/Kernel/ViewsIntegrationTest.php

```diff
@@ -11,7 +11,7 @@
  *
  * @group entity_hierarchy
  */
-class ViewsIntegrationTest extends EntityHierarchyKernelTestBase {
+abstract class ViewsIntegrationTest extends EntityHierarchyKernelTestBase {
```

Comment:
I think I'd prefer to keep these as classes but use ->markTestSkipped in each method so they show up in test runs